db/migrate/20191104192052_add_name_and_description_to_service_plans.rb

```diff
@@ -0,0 +1,6 @@
+class AddNameAndDescriptionToServicePlans < ActiveRecord::Migration[5.2]
+  def change
+    add_column :service_plans, :name, :string
```

Comment:
Was adding `name`, `description` due to a previous conversation, or were they missed. I see they are exposed in the openapi spec already.

Reply:
It was an oversight by me when creating the schema in #442

Reply:
I wonder if that change should be pulled into its own PR.

Reply:
Done! #467